### PR TITLE
AEA-2042 Handle signing service redirects in EPSAT

### PR DESCRIPTION
**Pull request by booshibooshi**

### CODE REVIEW

File: tool/site/server/src/services/communication/live-signing-client.ts [hunk 1]
```diff
@@ -19,7 +20,10 @@ export class LiveSigningClient implements SigningClient {
 
   async uploadSignatureRequest(prepareResponses: Parameters[]): Promise<any> {
     const baseUrl = this.getBaseUrl()
-    const url = `${baseUrl}/signaturerequest`
+    const stateJson = {prNumber: getPrNumber(CONFIG.basePath)}
+    const stateString = JSON.stringify(stateJson)
+    const state = Buffer.from(stateString, "utf-8").toString("base64")
```
Comment (JackRobinsonIW) [inline tool/site/server/src/services/communication/live-signing-client.ts:25]: again my knowledge gap but why do we need the `base64` parsing

**Review by JackRobinsonIW — APPROVED** [review]

File: tool/site/server/src/services/communication/live-signing-client.ts (hunk 1 above)
Comment (booshibooshi) [reply tool/site/server/src/services/communication/live-signing-client.ts:25]: it gets passed to a query string parameter

File: tool/site/server/src/app.ts [hunk 2]
```diff
@@ -139,7 +139,7 @@ function addViewRoutes(server: Hapi.Server) {
   server.route(addView("validate"))
   server.route(addView("search"))
   server.route(addView("prescribe/load"))
-  server.route(addView("prescribe/send"))
+  server.route(addView("prescribe/send", true))
```
[the remark this answers is not in the document]
Comment (booshibooshi) [reply tool/site/server/src/app.ts:142]: endpoint needs to be open as it's a callback from signing service and redirects through feature branches. there are safe guards for this route which check query parameters are valid etc, it matches the behaviour of the oauth callback route

**Pull request merged**